src/Components/LoginPage/SignPage.jsx

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import "./LoginStyles.css"
+import {SignBarForm} from "./LoginPageComponents/SignBarForm";
+import {FooterBlock} from "./LoginPageComponents/FooterBlock";
+
+
+export const SignPage = (props) => {
+    const initialState = {
+        signFormShowPassword: false,
+        signFormShowConfirmPassword: false
+    }
+
+    const [state, setState] = useState(initialState)
+
```

Comment:
Why have you united two states in one? That's not good practice

Reply:
I don't understand this question completely ((